hooks/post_push

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# https://docs.docker.com/docker-hub/builds/advanced/
+docker tag $IMAGE_NAME $DOCKER_REPO:$SOURCE_COMMIT
```

Comment:
@warrensbox - you can setup your Dockerhub account with auto-builds when a new tag is pushed to your repo. It is the simplest way to get a docker image to build on tags. It would be great to have an official Dockerhub image linked/built directly from the GitHub repo.